docs/BEST_PRACTICES.md

```diff
@@ -1,4 +1,17 @@
-## Deployment best practices
+## Deployment Guide
+
+As a quick start, you can utilize the available bash script which is capable of deploying both modules to a 
+Google Cloud Project. It requires to have an existing Kubernates cluster in Google Kubernates Engine. 
+Edit the configuration values in  deploy-to-gcp.sh and execute it.
+
+| Name | Description | Required | Default value (only for test environment) |
+| --- | --- | --- | --- |
+|`GCLOUD_PROJECT_ID` | Google Cloud project ID | YES | |
+|`GCLOUD_ZONE` | Google Cloud zone which cluster instances should be spawned [Zones](https://cloud.google.com/compute/docs/regions-zones#available). | YES | |
+|`GCLOUD_CLUSTER_NAME` | Existing Google Kubernetes Engine cluster name | YES | `adyen-demo` |
+|`TAG` | Indented [release version](https://github.com/commercetools/commercetools-adyen-integration/releases) of commercetools-adyen-integration | YES | `v5.0.0` |
```

Comment:
With default like v5.0.0, we need to manage it every change which is easy to forget, I would use the latest tag, which is default.

Reply:
c3293f785c5f2d7bc23617fb8238a97af0c75594